tests/index/server/devpi/entrypoint.sh

```diff
@@ -10,7 +10,7 @@ echo "==:> Starting server"
 devpi-server --host 0.0.0.0 --port 3141 &
 
 echo "==:> Waiting on server"
-timeout 10 ash -c 'while ! echo exit | nc localhost 3141; do sleep 1; done'
```

Comment:
I'm so upset that I don't have time to debug this. Essentially, the large Docker upgrade on Ubuntu 22.04 in [this release](https://github.com/actions/runner-images/releases/tag/ubuntu22%2F20240609.1) somehow makes it so that this no longer works. The connection to the port is never successful. I don't know why, this is a bummer.